Toolset/palettes/inspector/editors/com.livecode.pi.customprops.behavior.livecodescript

```diff
@@ -113,8 +113,12 @@ on editorResize
    send "groupResize kLabelSize, kControlGap" to group "Set buttons" of me
    
    # Use all space not taken by other elements for main array display
-   local tWidgetHeight
-   put the height of me - (4 * kControlGap + kLabelFieldHeight + kKeyFieldHeight + kValueFieldHeight + kSetButtonsHeight) into tWidgetHeight
+      local tWidgetHeight, tValueHeight
+   put 150 into tWidgetHeight -- or other value
+   put max (the height of this card - ( 12 * kControlGap + kLabelFieldHeight  \
```

Comment:
I'm not sure what this 12 is, but there was a random 4 beforehand so it's not your fault! However, if you do understand why this needs to be 12, could you put it into a suitably named constant?

Reply:
it was more controls and bottom kControlGaps and experimentation